guides-and-tutorials/running-a-signer/README.md

```diff
@@ -1,19 +1,46 @@
+
+The 3.0 binaries (and Docker images) are provided below. Note that if you do not upgrade ahead of the hard fork, your nodes will be dropped from the network.
+
+If you have previously been running a signer, you'll want to make sure you do the following:
+
+First, upgrade your node and signer to the latest version (listed below).
+
+For miners:
+If you are currently using the config value `[miner].activated_vrf_key_path`, the saved json file will need to be removed prior to restart (it will be recreated).
+You will also need to add a new key to the miner section:
+
+```
+[miner]
+mining_key = "private key" ## may be the same as [node].seed
+```
+
+**Important note**:
+Any unrecognized and uncommented config value _will_ cause the binary to panic.
+
+The output will specify which configuration key caused the panic (there may be more than one, but the message will only show the first unrecognized key) in the format:
+
+```
+$ stacks-node check-config --config ./Config.toml
+INFO [1729707354.217841] [testnet/stacks-node/src/main.rs:285] [main] stacks-node 3.0.0.0.0 (release/3.0.0.0.0:dd1ebe6, release build, linux [x86_64])
+INFO [1729707265.671604] [testnet/stacks-node/src/main.rs:318] [main] Loading config at path ./Config.toml
+WARN [1729707265.671992] [testnet/stacks-node/src/main.rs:325] [main] Invalid config file: Invalid toml: unknown field `foo`, expected one of `name`, `seed`, `deny_nodes`, `working_dir`, `rpc_bind`, `p2p_bind`, `p2p_address`, `data_url`, `bootstrap_node`, `local_peer_seed`, `miner`, `stacker`, `mock_mining`, `mock_mining_output_dir`, `mine_microblocks`, `microblock_frequency`, `max_microblocks`, `wait_time_for_microblocks`, `wait_time_for_blocks`, `next_initiative_delay`, `prometheus_bind`, `marf_cache_strategy`, `marf_defer_hashing`, `pox_sync_sample_secs`, `use_test_genesis_chainstate`, `always_use_affirmation_maps`, `require_affirmed_anchor_blocks`, `chain_liveness_poll_time_secs`, `stacker_dbs`, `fault_injection_block_push_fail_probability` for key `node` at line 20 column 1
+```
 
-{% hint style="info" %}
 #### Current Signer and Stacks Node Versions
 
 For quick reference, here are the current latest versions you'll want to be running as a signer. If you don't yet have your signer up and running, this guide will walk you through that.
 
-* Signer - [2.5.0.0.5.2](https://hub.docker.com/layers/blockstack/stacks-signer/2.5.0.0.5.2/images/sha256-fc29a7c22f236f91270fb1aa58cfb4dd8dcd6b1daa0812e16df0bdc7643cb6ac?context=explore)
-* Stacks Node - [2.5.0.0.7](https://hub.docker.com/layers/blockstack/stacks-core/2.5.0.0.7/images/sha256-71d3eb305b5c1b68cd44904a7bcd6e5f92542135a7218762cdf27a46acaff69b?context=explore)
-{% endhint %}
+- [Binaries](https://github.com/stacks-network/stacks-core/releases)
```

Comment:
maybe point to https://github.com/stacks-network/stacks-core/releases/latest  (which won't point to the signer binary specifically)?

Reply:
Done ✅